### PR TITLE
label-update

### PR DESCRIPTION
**Pull request by dakshaladia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,8 +57,12 @@ Node (22.9.0 or higher) and npm
    - **DO NOT COMMIT** this file. (thanks!)
 
 ## Using cypress testing
+Once done creating an end-to-end test in client/cypress/e2e,
 
-1. Once done creating an end-to-end test in client/cypress/e2e, view the test work using `npx open cypress` in the client folder. Make sure the application is already running.
+1. Stop the server. Inside server/app/, export mongo db creds:
+export MONGO_URI="mongodb+srv://<username>:<password>@animus.piv6r.mongodb.net/?retryWrites=true&w=majority&appName=Animus"
```
Comment (A-N1khil) [inline README.md:63]: I dont think we need this. Server will keep the auth on

File: client/cypress/e2e/DoctorPage.cy.js [hunk 2]
```diff
@@ -19,8 +19,8 @@ describe("Doctor Page - End-to-End Tests", () => {
       cy.get(".ant-select-dropdown").contains(role).click(); // Select the doctor role
   
       // Log in as the doctor
-      cy.get("input[id='login_email']").type(doctorEmail); // Adjust selector as needed
-      cy.get("input[id='login_password']").type(doctorPassword); // Adjust selector as needed
+      cy.get("input[id='email']").type(doctorEmail); // Adjust selector as needed
```
Comment (A-N1khil) [inline client/cypress/e2e/DoctorPage.cy.js]: @Rakshita-7 are these labels correct?

Comment (Rakshita-7) [reply client/cypress/e2e/DoctorPage.cy.js]: updated.

File: README.md (hunk 1 above)
Comment (dakshaladia) [reply README.md:63]: nahi bina iske creds ke liye run mein error deta hai

**Review by Rakshita-7 — APPROVED** [review]

**Pull request merged**